src/main/webapp/app/exercises/quiz/shared/questions/multiple-choice-question/multiple-choice-question.component.html

```diff
@@ -20,7 +20,7 @@ <h6>{{ 'artemisApp.quizQuestion.allOptions' | artemisTranslate }}</h6>
         <ng-template #renderedExplanation>
             <div [innerHTML]="renderedQuestion.explanation"></div>
         </ng-template>
-        <span class="label label-primary" [ngbPopover]="renderedExplanation" placement="right" triggers="mouseenter:mouseleave" *ngIf="question.explanation && showResult">
+        <span class="label label-primary" [ngbPopover]="renderedExplanation" placement="auto" triggers="mouseenter:mouseleave" *ngIf="question.explanation && showResult">
```

Comment:
i guess, default popover placement is `auto`

Reply:
I didn't touch these since for these the right orientation makes sense.

Or do you think all being auto is better here?

Reply:
I changed it in e1208b4